package.json

```diff
@@ -116,6 +116,7 @@
     "redux": "~4.0.1",
     "rimraf": "2.6.2",
     "rxjs": "~6.1.0",
+    "rxjs-compat": "^6.5.3",
```

Comment:
We removed this in a previous release to secure future updates.

Reply:
It's in devDependencies now, because I got build errors without it.